service/src/main/java/bio/terra/workspace/common/exception/UnhandledActivityLogException.java

```diff
@@ -0,0 +1,10 @@
+package bio.terra.workspace.common.exception;
```

Comment:
This also showed up in the region PR. Are they stacked?

Reply:
maybe accidentally. It's definitely related to this one tho.